basic-data-types/tuples.py

```diff
@@ -0,0 +1,12 @@
+# Tuples
+"""Task
+Given an integer, n, and n space-separated integers as input, create a tuple, t, of those n integers. Then compute and
+print the result of hash(t).
+
+Note: hash() is one of the functions in the __builtins__ module, so it need not be imported.
+"""
+
+if __name__ == '__main__':
+    n = int(input())
+    integer_list = tuple(map(int, input().split(' ')))
```

Comment:
no need to specify whitespace separator, it's a default one
https://docs.python.org/3/library/stdtypes.html#str.split
this can help to avoid casual starting/trailing whitespaces in the string